fakedata_source/v2/raw_crash/111/20180313/11107bd0-2d1c-4865-af09-80bc00180313

```diff
@@ -1 +1 @@
-{"MinidumpSha256Hash": "ae0666f161fed1a5dde998bbd0e140550d2da0db27db1d0e31e370f2bd366a57", "ProductName": "Firefox", "ReleaseChannel": "beta", "Version": "60.0b2", "dump_checksums": {"upload_file_minidump": "ae0666f161fed1a5dde998bbd0e140550d2da0db27db1d0e31e370f2bd366a57"}, "legacy_processing": 0, "submitted_timestamp": "2018-04-11T16:25:35.023214+00:00", "throttle_rate": 100, "timestamp": 1523463935.0180907, "type_tag": "bp", "uuid": "11107bd0-2d1c-4865-af09-80bc00180313"}
+{"MinidumpSha256Hash": "ae0666f161fed1a5dde998bbd0e140550d2da0db27db1d0e31e370f2bd366a57", "ProductName": "Firefox", "ReleaseChannel": "beta", "Version": "60.0b2", "collector_notes": [], "dump_checksums": {"upload_file_minidump": "ae0666f161fed1a5dde998bbd0e140550d2da0db27db1d0e31e370f2bd366a57"}, "payload": "multipart", "payload_compressed": "0", "submitted_timestamp": "2018-04-11T16:25:35.023214+00:00", "uuid": "11107bd0-2d1c-4865-af09-80bc00180313"}
```

Comment:
This reflects a bunch of changes to Antenna over the last 6 months. I added a few new fields and removed a bunch of legacy fields that weren't interesting.